app/build.gradle

```diff
@@ -4,6 +4,7 @@ apply plugin: 'com.neenbedankt.android-apt'
 apply plugin: 'hu.supercluster.paperwork'
 apply from: '../code_quality_tools/jacoco.gradle'
 apply plugin: 'com.frogermcs.androiddevmetrics'
+apply from: '../code_quality_tools/quality.gradle'
```

Comment:
Like @vanniktech said, java module has `/build/classes` instead of `/build/intermediates/classes`